Misc Fixes

AnturK opened the pull request:
Fixes #12128 Hopefully not breaking anything..
Fixes #12146 
Fixes #12193 
Makes doors open when bumped in janicart.

AnturK committed:
Fixes larva lying down rotation

AnturK committed:
Fixes pipe unbuckling

duncathan commented on code/game/machinery/machinery.dm:
```diff
@@ -284,6 +284,7 @@ Class Procs:
 
 //set_machine must be 0 if clicking the machinery doesn't bring up a dialog
 /obj/machinery/attack_hand(mob/user, check_power = 1, set_machine = 1)
+	..() // unbuckling etc
```
in most places like this, iirc, it's more like `if(..()) return 1`
maybe you should look at other unbuckling stuff to make sure? I could be wrong

AnturK replied:
Here i want to just call the side effects though. Not change the functionality.

duncathan replied:
well I'm pretty sure it's so that you don't unbuckle someone and activate a machine with one single click

AnturK committed:
Return for buckling procs
Makes unbuckling priority over removing bag in janicart
Makes doors bumped in janicart open.

freerealestate commented:
>Makes doors open when bumped in janicart.

thank god
